src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -222,13 +251,21 @@ std::vector<std::optional<HashTable<HashedType>>> build(const RadixContainer<Lef
   return hashtables;
 }
 
-template <typename T>
+template <typename T, typename HashedType, bool consider_null_values = false>
```

Comment:
see above - I'd avoid the default parameter (even more so as the default is set to something that will break things if you don't set it)